server/main.lua

```diff
@@ -12,21 +11,19 @@ local function generateId(table)
     return id
 end
 
-lib.callback.register('qbx_evidence:server:getPlayerStatus', function(_, targetSrc)
-    local player = exports.qbx_core:GetPlayer(targetSrc)
-    if not player or not next(playerStatus[targetSrc]) then return {} end
-    local status = playerStatus[targetSrc]
+lib.callback.register('qbx_evidence:server:setStatus', function(source, statuses)
+    local playerStatuses = Player(source).state.statuses or {}
```

Comment:
Why store the statuses in a table instead of individual state bags per status?

Reply:
Just keeping it organized. I figured
```lua
state.statuses = {
    gsr = expiration,
    widePupils = expiration,
    -- and so on = expiration
}
```
would be better than
```lua
state.gsr = expiration,
state.widePupils = expiration,
-- and so on
```

It also makes returning all active statuses for that player easier than trying to get each one individually

Reply:
I see the tradeoff. How big do you think this table would get? One of the drawbacks of keeping it in one table is needing to send all that data over the network to other players in one large chunk. The common use cases will be other players asking for a specific status. i.e. running a gsr test. If the statuses are in one table, the player running the test will need to read the entire table, versus reading one value.

Also, maybe we should prefix the state bag name with qbx to avoid potential conflicts? Feels like other scripts may use statuses as well since it's so generic.

Reply:
I definitely **do not** want to start the practice of doing `state.qbx_statuses.gsr` or whatever else. I'll just do it one by one lol

Reply:
You could have an array of all the status strings and then use a for loop whenever you want to iterate.

Reply:
Just another place someone who wants to add an additional status has to edit